style/_tabs.scss

```diff
@@ -33,7 +33,7 @@
     }
 
     // Disable focus indicator for pointer devices
-    &:not(.focus-visible) + label {
+    &:not(:focus-visible) + label {
```

Comment:
I'm almost certain this was a typo. It has the unfortunate consequence of turning off focus rings on tabs. Here's how. The `&:not(.focus-visible)` will match the `input` element because there is no `.focus-visible` class on it (I searched the code base thoroughly). So this selector might as well be `.sd-tab-set > input + label`, which will match the tab, set its outline to none.

I think this rule was intended to prevent an effect on touchscreen devices where you tap an element and it gets briefly highlighted (or darklighted).